tests/helper.py

```diff
@@ -87,3 +87,52 @@ def group_exp_then_log(group, tangent_vec, base_point):
     result = group.log(
         point=aux, base_point=base_point)
     return result
+
+
+def test_parallel_transport(space, metric, shape):
```

Comment:
Nice! Good idea - maybe we should think about refactoring more tests like this? Like the `_vectorization` tests...